src/routes/open/components/ReviewInformation/index.tsx

```diff
@@ -1,7 +1,8 @@
 import TableContainer from '@material-ui/core/TableContainer'
 import classNames from 'classnames'
 import React, { useEffect, useState } from 'react'
-
```

Comment:
Just a comment. This blank lines separating external dependencies and internal imports are ok

Reply:
we used to have a linting rule that divided the imports and sorted them. Not sure why it's no longer there.